src/core/Solana/resolveSolanaAddress.ts

```diff
@@ -0,0 +1,12 @@
+import { ChainType } from '@lifi/types'
+import { resolveUNSAddress } from '../EVM/uns/resolveUNSAddress.js'
```

Comment:
Oh, Solana ecosystem should not have any imports form EVM, it breaks dependency separation